cmd/loki/loki-local-config.yaml

```diff
@@ -12,7 +12,6 @@ common:
       rules_directory: /tmp/loki/rules
   replication_factor: 1
   ring:
-    instance_addr: 127.0.0.1
```

Comment:
Make sense to remove it from docker config. But i don't understand why from local-config as well? (we use this only to run in single node AFAIK).

Reply:
I agree, I'm going to remove this change.